src/com/chocohead/clm/blocks/MotorBlock.java

```diff
@@ -10,20 +10,15 @@
 import java.util.Locale;
 import java.util.Random;
 
-import com.jamieswhiteshirt.clotheslinefabric.api.NetworkManagerProvider;
-import com.jamieswhiteshirt.clotheslinefabric.api.NetworkNode;
-import com.jamieswhiteshirt.clotheslinefabric.api.NetworkState;
-import com.jamieswhiteshirt.clotheslinefabric.common.item.ClotheslineItems;
-
-import net.fabricmc.fabric.api.block.FabricBlockSettings;
-import net.fabricmc.fabric.api.tag.FabricItemTags;
-import net.minecraft.block.Block;
-import net.minecraft.block.BlockRenderLayer;
-import net.minecraft.block.BlockRenderType;
-import net.minecraft.block.BlockState;
-import net.minecraft.block.Blocks;
-import net.minecraft.block.Waterloggable;
-import net.minecraft.entity.VerticalEntityPosition;
+import com.jamieswhiteshirt.clothesline.api.NetworkManagerProvider;
+import com.jamieswhiteshirt.clothesline.api.NetworkNode;
+import com.jamieswhiteshirt.clothesline.api.NetworkState;
+import com.jamieswhiteshirt.clothesline.common.block.ClotheslineAnchorBlock;
+import com.jamieswhiteshirt.clothesline.common.item.ClotheslineItems;
+
+import net.fabricmc.fabric.api.object.builder.v1.block.FabricBlockSettings;
+import net.fabricmc.fabric.api.tool.attribute.v1.FabricToolTags;
+import net.minecraft.block.*;
```

Comment:
I'd like to avoid wildcard imports where possible (if anything just to avoid accidentally conflicting with a vanilla class's name)